internal/command/pause/worker.go

```diff
@@ -0,0 +1,72 @@
+package pause
+
+import (
+	"context"
+	"github.com/cirruslabs/orchard/pkg/client"
+	"github.com/spf13/cobra"
+	"time"
+)
+
+var wait uint64
+
+func newPauseVMCommand() *cobra.Command {
```

Comment:
```suggestion
func newPauseWorkerCommand() *cobra.Command {
```

Reply:
Renamed in https://github.com/cirruslabs/orchard/pull/73/commits/7fed7c2572967ec6143e86f9fd2fe712c5a1b41d.